src/devices/shelly.ts

```diff
@@ -30,4 +30,12 @@ export const definitions: DefinitionWithExtend[] = [
         description: "1PM Gen 4",
         extend: [m.onOff({powerOnBehavior: false}), m.electricityMeter({producedEnergy: true, acFrequency: true})],
     },
+    {
+        zigbeeModel: ["2PM"],
+        model: "2PM",
+        vendor: "Shelly",
+        description: "2PM Gen4",
+        extend: [m.deviceEndpoints({endpoints: {"1": 1, "239": 239}}), m.windowCovering({controls: ["lift", "tilt"]})],
```

Comment:
```suggestion
        extend: [m.windowCovering({controls: ["lift", "tilt"]})],
```

And remove `meta: {multiEndpoint: true},`, does it work with that?